edited "hilsa shad"

cmrn-rhi opened the pull request:
**- added Exact Synonyms:** ilish, ilisha, hilsa, hilsha, hilsa herring, jatka, ellis, palla fish, ilih, ilihi, ulla meen, modar, pallo machhi, pulasa, sboor, terubok, terubok umno
**- edited definition**
**- added image**

cmrn-rhi committed:
edited "hilsa shad"

- added Exact Synonyms: ilish, ilisha, hilsa, hilsha, hilsa herring, jatka, ellis, palla fish, ilih, ilihi, ulla meen, modar, pallo machhi, pulasa, sboor, terubok, terubok umno
- edited definition
- added image

ddooley commented:
Has problem of rdf:nodeid

ddooley commented:
has problem of nodeid.  Could try resaving "foodon-edit.owl" as "rdf/xml syntax"

cmrn-rhi committed:
Resaved foodon-edit.owl rdf/xml syntax

Resaved foodon-edit.owl file over top of the existing one via “save as …” “rdf/xml syntax”.
